Mathlib/Util/CountHeartbeats.lean

```diff
@@ -221,8 +222,9 @@ def countHeartbeatsLinter : Linter where run := withSetOptionIn fun stx ↦ do
   let mut msgs := #[]
   if [``Lean.Parser.Command.declaration, `lemma].contains stx.getKind then
     let s ← get
-    elabCommand (← `(command| #count_heartbeats in $(⟨stx⟩)))
-    msgs := (← get).messages.unreported.toArray.filter (·.severity != .error)
```

Comment:
note: I dropped `.filter (·.severity != .error)` because I don't see the necessity anymore and it was part of what hid the bug from the user initially.

Reply:
Agreed: before, it was meant to drop the error mentioning that the declaration already existed.  Now, it will highlight the few edge cases where there is an actual error!